app/masterhook.c

```diff
@@ -72,6 +72,15 @@ int drmModeAtomicCommit(int fd, drmModeAtomicReqPtr req,
     return ((typeof(drmModeAtomicCommit)*)dlsym(RTLD_NEXT, "drmModeAtomicCommit"))(fd, req, flags, user_data);
 }
 
+int drmIsMaster(int fd)
```

Comment:
I don't think this will compile on with the older libdrm headers, because you won't have a declaration of `drmIsMaster()` prior to its use above. You can just move this function to the top of this file.

Reply:
Ah, yes that's embarrassing. Moved the function above the calling ones. Thanks!